cpp/CMakeLists.txt

```diff
@@ -780,6 +780,10 @@ install(DIRECTORY ${CUDF_SOURCE_DIR}/include/cudf ${CUDF_SOURCE_DIR}/include/cud
                   ${CUDF_SOURCE_DIR}/include/nvtext DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
 )
 
+if(DLPACK_INCLUDE_DIR)
+  install(DIRECTORY ${DLPACK_INCLUDE_DIR}/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+endif()
```

Comment:
This was a temporary workaround, we should instead promote `dlpack_SOURCE_DIR` to the parent scope [here](https://github.com/rapidsai/cudf/blob/a926c52d58c08657f8d437210ce31fddeaa868e7/cpp/cmake/thirdparty/get_dlpack.cmake#L29) and only install this dir if that variable is set.